backend/api/graphql/schema.py

```diff
@@ -0,0 +1,18 @@
+from graphene import ObjectType, Schema
+
+from source.mutations.DeleteSourceMutation import DeleteSourceMutation
+from source.queries import SourceQueries
+
+from source.mutations.UpdateOrCreateSourceMutation import UpdateOrCreateSourceMutation
+
+
+class Query(SourceQueries, ObjectType):
```

Comment:
Not everyone is a fan of inheritance like this. We can also write all the queries (and their resolvers here), but that usually ends up being a very long list.